packages/block-library/src/query/view.js

```diff
@@ -62,6 +62,7 @@ store( {
 								: '' );
 
 						context.core.query.animation = 'finish';
+						context.core.query.url = ref.href;
```

Comment:
I'm curious. What's the advantage of using the context to store the link you want to prefetch and use an effect callback, instead of directly calling `prefetch`?

```suggestion
						actions.core.query.prefetch( ref );
```

This way, we could also get rid of `effects.core.query.prefetch`.

Reply:
Oh, because you only want to do that for the Pagination Next and Pagination Previous blocks, am I right?

Reply:
We want to do the prefetch of the new `href` of Next and Previous elements, yes. I'm just storing the new URL to trigger those effects again.